brayns/common/Timer.h

```diff
@@ -24,50 +24,53 @@
 
 namespace brayns
 {
+using clock = std::chrono::high_resolution_clock;
+
 /** Simple timer class to measure time spent in a portion of the code */
 class Timer
 {
 public:
-    Timer() = default;
+    Timer();
 
     /** (Re)Start the timer at 'now' */
-    void start() { _startTime = clock::now(); }
-    /** @return the elapsed time in seconds since the last start(). */
-    double elapsed() const
-    {
-        return std::chrono::duration<double>{clock::now() - _startTime}.count();
-    }
+    void start();
 
     /** Stops the timer and records the interval + a smoothed value over time*/
-    void stop()
-    {
-        _milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(
-                            clock::now() - _startTime)
-                            .count();
-        _smoothNom = _smoothNom * _smoothingFactor + _milliseconds / 1000.0;
-        _smoothDen = _smoothDen * _smoothingFactor + 1.f;
-    }
+    void stop();
+
+    /** @return the elapsed time in seconds since the last start(). */
+    double elapsed() const;
 
     /** @return last interval from start() to stop() in milliseconds. */
-    int64_t milliseconds() const { return _milliseconds; }
+    int64_t milliseconds() const;
+
     /** @return last interval from start() to stop() in seconds. */
-    double seconds() const { return _milliseconds / 1000.0; }
+    double seconds() const;
+
     /**
      * @return last interval from start() to stop() in per seconds, e.g. for
      * frame per seconds
      */
-    double perSecond() const { return 1000.0 / _milliseconds; }
+    double perSecond() const;
```

Comment:
Remove functions not used? (I'm guessing that `seconds()` and perSecond()` aren't`

Reply:
I moved this to another PR.